server/routes/prisonerImages.test.ts

```diff
@@ -36,4 +41,13 @@ describe('GET /prisoner-images/:imageId.jpeg', () => {
         expect(res.text).toEqual('image 123 data')
       })
   })
+
+  it('does not trigger getUser()', () => {
+    return request(app)
+      .get(`/prisoner-images/123.jpeg`)
+      .expect('Content-Type', /images\/jpeg/)
+      .expect(res => {
+        expect(userService.getUser).toBeCalledTimes(0)
```

Comment:
Unit test to check that these calls don't happen when getting an image.

(will fail if moving the `/` route above the images route [here](https://github.com/ministryofjustice/hmpps-incentives-ui/pull/25/files#diff-47e00f44a4610e3f7bcaad8433c7bccf2ae050a627a3399de9553d7c910cd7b0R94))

Reply:
this sounds fair enough, but given that the routing for tests is setup in a different place from the normal app's routing, it's still possible to accidentally change the real router and have this test pass

Reply:
You're right. I think at the moment this is a risk (not specific of this test)

I don't have a solution for this yet, but I think we may want to experiment with ways to improve separately.

Maybe we could experiment with the way routing is wired together at some point (e.g. have only one router mounted into `/` so that the `appSetup` module doesn't need to be updated. But when I played with this idea briefly everything broke 😬